cpanfile

```diff
@@ -7,6 +7,9 @@ requires 'XML::Simple';
 requires 'IO::File';
 requires 'List::Util';
 requires 'LWP::Simple';
+requires 'Selenium::Remote::Driver';
+requires 'Selenium::Chrome';
+requires 'Selenium::Waiter';
```

Comment:
Hm, I'm not so sure about this added dependency. Is it too heavy weight in some situations or ok in general?